vars/createSha512Checksums.groovy

```diff
@@ -11,8 +11,11 @@ Closure call() {
                 if (file.endsWith(fileType)) {
                     echo("Creating sha for ${file}")
                     final sha512 = sh(script: "sha512sum ${file}", returnStdout: true).split()
-                    // Reading the first index of array since sha512sum return [shasum, filename] as output
-                    writeFile file: "${file}.sha512", text: "${sha512[0]}"
+                    //sha512 is an array [shasum, filename]
+                    final basename = sh(script: "basename ${sha512[1]}", returnStdout: true).split()
```

Comment:
Why do you need to split for the output?
Isnt `basename` will only return one entry which is the filename.zip?
Thanks.

Reply:
The first `sha512` would contain `sha /path/to/file.zip`
The basname is extracts `file.zip` from `sha /path/to/file.zip`

Reply:
I 

> Also, pytest issues.

Did you mean groovy tests? I updated those for shafile changes to be as robust as possible

Reply:
Then you dont need the second split for basename.